backend/coins/btc/handlers/handlers.go

```diff
@@ -53,6 +53,7 @@ func NewHandlers(
 	handleFunc("/init", handlers.postInit).Methods("POST")
 	handleFunc("/status", handlers.getAccountStatus).Methods("GET")
 	handleFunc("/transactions", handlers.ensureAccountInitialized(handlers.getAccountTransactions)).Methods("GET")
+	handleFunc("/transaction", handlers.ensureAccountInitialized(handlers.getTransaction)).Methods("GET")
```

Comment:
consistency: add or remove `Account` in both function names

Reply:
Didn't use `getAccountTransaction` because in my mind: 
`getAccountTransactions` -> retrieve all the transaction of a certain account
`getTransaction` -> retrieve a specific transaction (by ID)

I wouldn't remove `Account` from  `getAccountTransactions`, but I see no problems in making `getAccountTransaction`